src/dex-ui/services/DexService/dao/DAOService.ts

```diff
@@ -450,21 +450,32 @@ export async function fetchGovernanceDAOLogs(governors: DAOProposalGovernors): P
   const getTokenTransferDetailsFromHexData = (data: string | undefined) => {
     if (isNil(data)) return { ...DefaultTokenTransferDetails };
     const abiCoder = ethers.utils.defaultAbiCoder;
-    const parsedData = abiCoder.decode(
-      [
-        "address transferFromAccount",
-        "address transferToAccount",
-        "address tokenToTransfer",
-        "uint256 transferTokenAmount",
-      ],
-      data
-    );
-    return {
-      transferFromAccount: solidityAddressToTokenIdString(parsedData.transferFromAccount),
-      transferToAccount: solidityAddressToTokenIdString(parsedData.transferToAccount),
-      tokenToTransfer: solidityAddressToTokenIdString(parsedData.tokenToTransfer),
-      transferTokenAmount: convertEthersBigNumberToBigNumberJS(parsedData.transferTokenAmount).toNumber(),
-    };
+    const operationTypeData = ethers.utils.defaultAbiCoder.decode(["uint256 operationType"], data);
+    const operationType = convertEthersBigNumberToBigNumberJS(operationTypeData.operationType).toNumber();
+    if (operationType === 1) {
```

Comment:
please use Defined enum or we can create one to differentiate between the proposals `enum`

Reply:
Done.